README.md

```diff
@@ -240,6 +240,7 @@ Heat meter Techem Compact V (compact5) (non-standard protocol)
 Heat meter Techem Vario 4 (vario451) (non-standard protocol)
 Heat meter Kamstrup Multical 302 (multical302) (in C1 mode, please open issue for T1 mode)
 Heat and Cooling meter Kamstrup Multical 403 (multical403) (in C1 mode)
+Heat and Cooling meter Kamstrup Multical 603 (multical403) (in C1 mode) (work in progress)
```

Comment:
```suggestion
Heat and Cooling meter Kamstrup Multical 603 (multical603) (in C1 mode) (work in progress)
```

Reply:
Good catch - not sure if its me or @weetmuts who should accept the suggested change.